filesystem/usr/local/share/landoop/etc/supervisord.d/99-supervisord-running-sample-data.conf

```diff
@@ -72,11 +72,7 @@ startretries=1
 
 [program:lenses-processor]
 user=nobody
-command=bash -c 'for ((i=0;i<60;i++)); do sleep 3; lenses-cli --timeout 3s --user admin --pass admin --host http://localhost:$WEB_PORT processors && { sleep 5; break; }; done; lenses-cli --user admin --pass admin --host http://localhost:$WEB_PORT processor create --name=filter_fast_vessels --sql="SET autocreate=true;
-        INSERT INTO fast_vessel_processor
-        SELECT MMSI, Speed, Longitude AS Long, Latitude AS Lat, \`Timestamp\`
-        FROM sea_vessel_position_reports
-        WHERE Speed > 10"'
+command=bash -c '/usr/local/share/landoop/sample-data/lenses-processor.sh'
```

Comment:
You don't need `bash -c` since you have an executable script. :p

`command=/usr/local/share/landoop/sample-data/lenses-processor.sh`